.github/workflows/install_test.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+
+name: Install Test
+jobs:
+  install_evalml:
+    name: ${{ matrix.python_version }} install EvalML
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: ["3.7", "3.8", "3.9"]
+    steps:
+      - name: Set up python ${{ matrix.python_version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install EvalML and update checker
+        run: |
+          pip config --site set global.progress_bar off
```

Comment:
@jeff-hernandez why is this necessary?

Reply:
It helps to make the installation process more readable

Reply:
That's neat. Do you think we should do this in all our CI? Is that what featuretools, woodwork and compose do?

Reply:
Yeah, it makes the package installation easier to follow. We also do this in featuretools, woodwork, and compose.